rundeckapp/grails-spa/packages/ui-trellis/src/library/components/plugins/PluginDetails.vue

```diff
@@ -1,13 +1,25 @@
 <template>
-  <span v-if="showDescription" :class="descriptionCss" style="margin-left: 5px">
+  <span
+    v-if="showDescription && !inlineDescription"
+    :class="descriptionCss"
+    style="margin-left: 5px"
+    data-testid="block-description"
+  >
     {{ shortDescription }}
   </span>
   <details
     v-if="showDescription && showExtended && extraDescription"
-    class="more-info details-reset"
+    class="more-info"
     :class="extendedCss"
   >
     <summary>
+      <span
+        v-if="showDescription && inlineDescription"
+        :class="descriptionCss"
+        data-testid="inline-description"
+      >
+        {{ shortDescription }}
+      </span>
       {{ $t("more") }}
```

Comment:
the "More" text here kind of blends into the shortDescription, since there is no style difference.  The earlier implementation used a btn-xs with a blue color for this text.

I think perhaps moving the "More..." inside the `.more-indicator-verbiage` might make sense, it would colorize it the same as the chevron.  We could also put "Less" text inside the  `.less-indicator-verbiage`, which would be closer to the original.

what do you think?

old:
<img width="328" alt="Screenshot 2024-08-14 at 11 34 55 AM" src="https://github.com/user-attachments/assets/73d08104-8a38-4597-b50f-eb103f9eea3e">
<img width="301" alt="Screenshot 2024-08-14 at 11 34 58 AM" src="https://github.com/user-attachments/assets/938880f6-8767-448f-9ac7-08bdf58557f7">

this change:
<img width="275" alt="Screenshot 2024-08-14 at 11 35 30 AM" src="https://github.com/user-attachments/assets/e8b95c11-3301-4aa6-995e-c02a0be71f07">
<img width="243" alt="Screenshot 2024-08-14 at 11 35 33 AM" src="https://github.com/user-attachments/assets/4ba1418e-3457-405d-a660-b893ae345c6a">

after putting text inside the more-indicator-* sections:

<img width="175" alt="Screenshot 2024-08-14 at 11 36 47 AM" src="https://github.com/user-attachments/assets/160744ff-cbd2-4e27-834c-ad49ad3fa382">
<img width="165" alt="Screenshot 2024-08-14 at 11 36 51 AM" src="https://github.com/user-attachments/assets/0a95929d-3c0d-499e-8ad5-8f60b0dfd943">

Reply:
Good idea, think that the intent will be clearer this way!